tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/JSDependencyManifest.scala

```diff
@@ -90,8 +91,8 @@ object JSDependencyManifest {
     def hasConflict(x: FlatJSDependency, y: FlatJSDependency) = (
       x.commonJSName.isDefined &&
       y.commonJSName.isDefined &&
-      (x.resourceName == y.resourceName ^
-       x.commonJSName == y.commonJSName)
+      (x.relPath == y.relPath ^
```

Comment:
This needs proper prefix matching. Otherwise this will fail:

```
"jasmine.js" commonJSName "jasmine"
"foo/bar/jasmine.js" commonJSName "jasmine"
```

Even if they refer to the same file.